pkg/provider/azure_loadbalancer_repo_test.go

```diff
@@ -54,7 +55,7 @@ func TestListManagedLBs(t *testing.T) {
 	tests := []struct {
 		existingLBs, expectedLBs []network.LoadBalancer
 		callTimes                int
-		multiSLBConfigs          []MultipleStandardLoadBalancerConfiguration
+		multiSLBConfigs          []ratelimitconfig.MultipleStandardLoadBalancerConfiguration
```

Comment:
In azure_loadbalancer.go, this is **azureconfig**.MultipleStandard....

Reply:
I will rename it to azureconfig. Thanks!